alicloud/alicloud_cs_kubernetes_cluster.go

```diff
@@ -0,0 +1,487 @@
+package alicloud
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/aliyun/alibaba-cloud-sdk-go/sdk/errors"
+	"github.com/aliyun/alibaba-cloud-sdk-go/sdk/requests"
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/cs"
+
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAlicloudCsKubernetesCluster(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "alicloud_cs_kubernetes_cluster",
+		Description: "Alicloud Container Service Kubernetes Cluster",
+		List: &plugin.ListConfig{
+			Hydrate: listCsKubernetesClusters,
+		},
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.AllColumns([]string{"cluster_id", "region"}),
+			Hydrate:    getCsKubernetesCluster,
+		},
+		GetMatrixItem: BuildRegionList,
+		Columns: []*plugin.Column{
```

Comment:
Please arrange the column in following order:
1. Add all the important columns ( max. 4-5 columns) at the top
2. Add rest of the columns in alphabetical order
3. Add the JSON type columns at the end